smartrouter/image.yaml

```diff
@@ -0,0 +1,27 @@
+name: "jboss-bpmsuite-7/bpmsuite70-smartrouter"
+description: "Red Hat JBoss BPM Suite 7.0 Smart Router container image"
+version: "7.0.0"
+from: "jboss/openjdk18-rhel7:1.0"
+user: 185
+envs:
+    - name: "JBOSS_PRODUCT"
+      value: "bpmsuite"
```

Comment:
We need to find if this is the best value for this environment variable. What we're shipping is in fact the smart router feature, not the full bpmsuite. Is that correct? If yes - we need to define some guidelines about this.